HoloRepositoryDemoApplication/Assets/HoloRpository/Scripts/StorageConnectionServer.cs

```diff
@@ -0,0 +1,256 @@
+
+            hologramList.Clear();
+            if (WebRequestReturnData != null)
+            {
+                JSONNode InitialJsonData = JSON.Parse(WebRequestReturnData);
+                JSONArray JsonArray = InitialJsonData.AsArray;
+                foreach (JSONNode HologramJson in JsonArray)
+                {
+                    Hologram hologram = JsonToHologram(HologramJson);
+                    if (hologram.hid != null)
+                    {
+                        hologramList.Add(hologram);
+                    }
+                }
+            }
+        }
+
+        public static IEnumerator GetHologram(Hologram hologram, string HolgramID)
+        {
+            string GetHologramUri = BaseUri + "/holograms/" + HolgramID;
+            yield return GetRequest(GetHologramUri);
+            try
+            {
+                JSONNode HologramJson = JSON.Parse(WebRequestReturnData);
+                Hologram Hologram = JsonToHologram(HologramJson);
+                CopyProperties(Hologram, hologram);
+            }
+            catch (Exception e)
+            {
+                Debug.Log("Failed to get the hologram from server! \n[Error message]: " + e.Message);
+            }               
+        }
+
+        public static async void LoadHologram(string HologramID)
+        {
+            WebRequestReturnData = null;
+            //string GetHologramUri = BaseUri + "/holograms/" + HologramID + "/download";
+            string GetHologramUri = "https://holoblob.blob.core.windows.net/test/DamagedHelmet-18486331-5441-4271-8169-fcac6b7d8c29.glb";      
+
+            Response response = new Response();
+            try
+            {
+                response = await Rest.GetAsync(GetHologramUri);
+            }
+            catch (Exception e)
+            {
+                Debug.LogError(e.Message);
+            }
+
+            if (!response.Successful)
+            {
+                Debug.LogError($"Failed to get glb model from {GetHologramUri}");
+                return;
+            }
+
+            var gltfObject = GltfUtility.GetGltfObjectFromGlb(response.ResponseData);
+
+            try
+            {
+                GameObject loadedObject = await gltfObject.ConstructAsync();
+                ModelSetting.Initialize(loadedObject);
+            }
+            catch (Exception e)
+            {
+                Debug.LogError($"{e.Message}\n{e.StackTrace}");
+                return;
+            }
+        }
+        #endregion Public Method
+
+        #region Private Common Method
+        private static IEnumerator GetRequest(string uri)
+        {
+            using (UnityWebRequest webRequest = UnityWebRequest.Get(uri))
+            {
+                yield return webRequest.SendWebRequest();
+                WebRequestReturnData = null;
+                if (webRequest.isNetworkError)
+                {
+                    Debug.Log("Web request Error! [Error message]: " + webRequest.error);
```

Comment:
Inconsistent logging, should always be Debug.LogError (not gonna write this comment any more, you could just Search and replace it everywhere (at least for all error logs)